crates/utils/src/errors.rs

```diff
@@ -0,0 +1,14 @@
+#[derive(Debug)]
+pub enum ApiError {
+    ApiInitialisationFailed(String),
```

Comment:
Here would be better to keep `tonic::Error`, don't need to convert it to string.